src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -116,15 +122,18 @@ const SelectPanelHeader: React.FC<React.PropsWithChildren> = ({children, ...prop
         {slots.heading}
         <Box>
           {/* Will not need tooltip after https://github.com/primer/react/issues/2008 */}
-          <Tooltip text="Clear selection" direction="s" onClick={onClearSelection}>
-            <IconButton type="button" variant="invisible" icon={ClearIcon} aria-label="Clear selection" />
-          </Tooltip>
+          {onClearSelection ? (
+            <Tooltip text="Clear selection" direction="s" onClick={onClearSelection}>
```

Comment:
Hopefully we can update here to use [the new Tooltip](https://github.com/primer/react/pull/3394) soon 🤞🏻